tests/containers/podman_netavark.pm

```diff
@@ -0,0 +1,143 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman, netavark, aardvark
+# Summary: Test podman netavark network backend
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp is_transactional);
+use containers::utils qw(get_podman_version);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+use Utils::Systemd qw(systemctl);
+
+sub remove_subtest_setup {
+    assert_script_run("podman container rm -af");
+    assert_script_run("podman network prune -f");
+    assert_script_run("podman network ls");
+    assert_script_run("podman ps -a");
+}
+
+sub _cleanup {
+    my $podman = shift->containers_factory('podman');
+    select_console 'log-console';
+    remove_subtest_setup;
+    script_run('rm -rf /etc/containers/containers.conf');
+    $podman->cleanup_system_host();
```

Comment:
isn't this function already what your `remove_subtest_setup` does?

Reply:
The former does not clean up the network settings.